src/OrchardCore/OrchardCore.Navigation.Core/AdminMenuNavigationProvider.cs

```diff
@@ -0,0 +1,9 @@
+namespace OrchardCore.Navigation;
+
+public abstract class AdminMenuNavigationProvider : NamedNavigationProvider
```

Comment:
`AdminNavigationProvider` instead

Reply:
@hishamco It was `AdminNavigationProvider` but Seb requested to change it to `AdminMenuNavigationProvider` so we'll keep it as`AdminMenuNavigationProvider`

Reply:
I disagree with Seb :)

Reply:
@sebastienros I am also okay with the original `AdminNavigationProvider`. Are you?

Reply:
@hishamco I renamed this.